### ISSUE TITLE
Docker Desktop V3.3.3 does not start (i.a.)

### ISSUE THREAD

**Issue by tjgalama** [docker/for-win#11191]
- [x] I have tried with the latest version of Docker Desktop
  - [x] I have disabled experimental features
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 
		 DB1844A9-A97D-4ACB-93B9-E42549F6D435/20210512130911

I first had DD running successfully on V3.3.1 (or v3.3.2) for Windows and Linux containers(WSL2) combined with local Kubernetes

After updating to V3.3.3 I got problem that the system agent Popup only displayed commands 'Dashboard' & 'Settings'  
![DockerPopupIssue](https://user-images.githubusercontent.com/504789/117991325-3ac0f300-b33e-11eb-9b52-1a0c7a954f47.png)

Also after a reboot / restart,  docker remained in a not-running state
After a 'reset to factory Defaults', Docker started successfully. However, after a system restart, Docker again got stock in state starting. I didn't have this problems in a previous version (V3.3.1 or 3.3.2). Trying to get it working again I un-installed V3.3.3 and reinstalled V3.3.1. Even then, docker would still not start and also the popup only shows the 2 commands. Looking back the it might have been better to reboot before the v3.3.1 install. Anyway, I reinstalled V3.3.3 and nothing changed. 

So next I started recording the actions explicitly:
- I uninstalled V3.3.3 and restarted the system.
- I re-installed V3.3.3
- Start Docker-desktop: 
	> docker remained in state 'stopped', (also agent popup, only displayed commands 'Dashboard' & 'Settings' )
- Rebooted the system, Started Docker-Desktop:
	> Again docker remained in state 'stopped', (also agent popup, only displayed commands 'Dashboard' & 'Settings' )
- Activate 'Reset to Factory defaults'
	> docker Started up to running (yesss). The  agent popup, still only displayed commands 'Dashboard' & 'Settings'.
- executed docker run on container 'docker/getting-started' (which succeeded !!)
- Enabled WLS Kubernetes v1.19.7 and hit Apply & restart
	> After a while Docker and Kubernetes where in a state Running (Yesss, again ) .
- Rebooted the system and start docker desktop:
	> Again both Docker & Kubernetes remained in state stopped 
	The Docker-Desktop view shortly showed a red banner notifying me that I had to restart docker.
	However, I am unable to quit decently since the agent's popup only shows 2 commands and button 'Restart Docker Desktop' is disabled
- Uploaded Diagnostic's

### Expected behavior
After starting Docker desktop, I expect:
1: Docker & Kubernetes going to running state
2: I expect the system agent's popup to show all commands just like in previous versions

### Information
  - Windows Version: Windows 10 Pro -version 20H2
  - Docker Desktop Version: 3.3.3
  - WSL2 or Hyper-V backend?   WSL2
  - Are you running inside a virtualized Windows e.g. on a cloud server or a VM: no
  - Also I have BitLocker activated. therefore for Windows containers I use the workaround mention [here](https://github.com/docker/for-win/issues/1297#issuecomment-441248003)

**Comment by gintuoklis** [comment]
I'm experiencing the same issue. After updating to 3.3.3 docker Desktop does not start at all. Tried restarting machine, but it did not help. Not using WSL2, just windows containers.

**Comment by ShabnamSarupBR** [comment]
I have the same problem - hoping this gets resolved soon

**Comment by tjgalama** [comment]
@gintuoklis; I just ran docker diagnostics but failed to recognize something that could lead to solving my situation.

**Comment by tiagomartins91** [comment]
The same issue with WSL2 on Windows, docker does not start.

**Comment by tkristolaitis-alna** [comment]
@gintuoklis copying my config from "C:\ProgramData\Docker\config\daemon.json" to "C:\Users\myuser\.docker\daemon.json" solved it for me as well.
Thanks alot.

**Comment by stephen-turner** [comment]
I think this is a dupe of #11047, so I'm going to close this issue and you can follow there for updates. In particular please see my latest comment there (https://github.com/docker/for-win/issues/11047#issuecomment-841136100) for something to try.

**Comment by tjgalama** [comment]
@gintuoklis the folder  "C:\ProgramData\Docker" does not contain sub folder "config" on my system.

This  morning I started up my laptop. After a while I clicked Docker-Desktop and it just started (incl. docker+kubernetes). Also the pop-up showed all it commands. I ran the following commands just as a short test:
```
docker run -d -p 80:80 docker/getting-started
kubectl create namespace keda
helm install keda kedacore/keda --version 2.2.0 --namespace keda --set nodeSelector."kubernetes\.io/os"=linux
kubectl get pods -A
```
Additionally, I added a diagnostic to allow difference analysis my first post (ID:DB1844A9-A97D-4ACB-93B9-E42549F6D435/20210514091542). Being skeptical,  I rebooted my system and again started docker successfully. 

--I have no idea what happened --
